views/about.ejs

```diff
@@ -0,0 +1,223 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+    <!---This is The color mode of the page-->
+    <script src="public/js/color-modes.js"></script>
+    <!--End of color mode-->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Bootstrap demo</title>
+    <link rel="stylesheet" href="public/css/dashboard.css">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-T3c6CoIi6uLrA9TneNEoa7RxnatzjcDSCmG1MXxSR1GAsXEV/Dwwykc2MPK8M2HN" crossorigin="anonymous">
+</head>
+
+<body>
+<style>
```

Comment:
these styles look like its similar to what's on the bootstrapindex.ejs.
do not repeat yourself.

you can put all this in the dashboard.css to avoid repeating